### PR TITLE
Update version of hca cli

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose
This version of the hca cli includes a fix to uploading large files

---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- No changes.
---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

- No instructions.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied code style guidelines:
    - [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) Python code.
    - Mint WDL style guide for WDL.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

**Comment by samanehsan** [comment]
retest this please

File: adapter_pipelines/Optimus/adapter.wdl [hunk 1]
```diff
@@ -139,7 +139,7 @@ workflow AdapterOptimus {
   Int max_cromwell_retries = 0
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.48.2"
+  String pipeline_tools_version = "se-update-hca-cli-version"
```
Comment (samanehsan) [inline adapter_pipelines/Optimus/adapter.wdl]: Will change this version before merging

File: setup.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
           'arrow>=0.12.1',
           'google-auth>=1.6.1,<2',
           'google-cloud-storage>=1.10.0,<2',
-          'hca>=4.5.0,<5',
+          'hca>=5.1.0',
```
Comment (rexwangcc) [inline setup.py]: How about ,<6 as well?

Comment (samanehsan) [reply setup.py]: Done!

**Pull request merged**